src/pages/developers/index.tsx

```diff
@@ -0,0 +1,470 @@
+  <Box flex="1 1 33%" mb={6} mr={8} w="full" {...props} />
+)
+const RightColumn = (props: ChildOnlyProp) => (
+  <Box flex="1 1 33%" mb={6} mr={0} w="full" {...props} />
+)
+const IntroColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 33%"
+    mb={6}
+    mt={{ base: 0, lg: 32 }}
+    mr={{ base: 0, sm: 8 }}
+    w="full"
+    {...props}
+  />
+)
+
+const StyledCard = (props: ICardProps) => {
+  const tableBoxShadow = useColorModeValue("tableBox.light", "tableBox.dark")
+
+  return (
+    <Card
+      boxShadow={tableBoxShadow}
+      m={4}
+      p={6}
+      {...props}
+      _hover={{
+        borderRadius: "4px",
+        boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
+        background: "tableBackgroundHover",
+        transition: "transform 0.1s",
+        transform: "scale(1.02)",
+      }}
+    />
+  )
+}
+
+const StyledCallout = chakra(Callout, {
+  baseStyle: {
+    flex: { base: "auto", md: "1 1 416px" },
+  },
+})
+export const getStaticProps: GetStaticProps = async (
+  context
+) => {
+  const { locale } = context
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPath('/developers')
+  const lastDeployDate = getLastDeployDate()
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}
+
+interface IDevelopersPath {
+  emoji: string
+  title: ReactNode
+  description: ReactNode
+  url: string
+  button: ReactNode
+}
+
+const paths: Array<IDevelopersPath> = [
+  {
+    emoji: ":woman_student:",
+    title: <Translation id="page-developers-index:page-developers-learn" />,
+    description: <Translation id="page-developers-index:page-developers-learn-desc" />,
+    url: "/developers/docs/",
+    button: <Translation id="page-developers-index:page-developers-read-docs" />,
+  },
+  {
+    emoji: ":woman_teacher:",
+    title: <Translation id="page-developers-index:page-developers-learn-tutorials" />,
+    description: <Translation id="page-developers-index:page-developers-learn-tutorials-desc" />,
+    url: "/developers/tutorials/",
+    button: <Translation id="page-developers-index:page-developers-learn-tutorials-cta" />,
+  },
+  {
+    emoji: ":woman_scientist:",
+    title: <Translation id="page-developers-index:page-developers-start" />,
+    description: <Translation id="page-developers-index:page-developers-start-desc" />,
+    url: "/developers/learning-tools/",
+    button: <Translation id="page-developers-index:page-developers-play-code" />,
+  },
+  {
+    emoji: ":construction_worker:",
+    title: <Translation id="page-developers-index:page-developers-set-up" />,
+    description: <Translation id="page-developers-index:page-developers-setup-desc" />,
+    url: "/developers/local-environment/",
+    button: <Translation id="page-developers-index:page-developers-choose-stack" />,
+  },
+]
+
+const DevelopersPage = ({ lastDeployDate }) => {
```

Comment:
Wondering why no prop typing here, or no use of `NextPageWithLayout`?

Reply:
Correct. Following the official docs, we could use (which is what I'm using in the Home page PR) to avoid repeating the props declaration
```ts
const DevelopersPage = ({ lastDeployDate }: InferGetStaticPropsType<typeof getStaticProps>)) => {
```